# an array of vpPoint
visp_camera_calibration/CalibPoint[] points
